.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: "\U0001F41BBug report"
+about: Create a report to help us improve Optuna.
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Bug
+
+Please write a clear and concise description of what the bug is.
+
+## Expected behavior
+
+Please write a clear and concise description of what you expected to happen.
+
+## Environment 
+
+- Optuna version:
+- Python version:
+- OS:
+- Machine Learning library to be optimized:
+
+## Error messages, stack traces, or logs
+
+```
+# error messages, stack traces, or logs
+```
+
+## Steps to reproduce
+
+1.
+2.
+3.
```

Comment:
I think we can use a numbered list of MarkDown as follows:

Source
```
1.
1.
1.
```

Rendered

1.
1.
1.

Do you have any specific reason to use the current format?

Reply:
> the current format is more intuitive

I see. Please keep as it is.
(I was just curious because I'm used to MarkDown's numbered list.)